### ISSUE TITLE
develop branch: (some) modules fails in function register

### ISSUE THREAD

**Issue by khassel** [MichMich/MagicMirror#2011]
The changes in b9d19cfcb4445f3a3c829dce16709f692fc96aa5 breaks (some or all foreign?) modules.

In my setup MMM-Strava and MMM-DarkSkyForecast are broken.

````
ReferenceError: global is not defined
register https://mm.karsten13.de/js/module.js:469
<anonym> https://mm.karsten13.de/modules/MMM-DarkSkyForecast//MMM-DarkSkyForecast.js:41
````

**Comment by khassel** [comment]
@rejas may you can take a look this? Thanks.

**Comment by rejas** [comment]
Was able to reproduce and fix, so could you check if the PR #2012 fixes it on your side too?

**Comment by khassel** [comment]
yes! Thanks.

**Comment by MichMich** [comment]
Merged #2012. Can I close this issue, @khassel?
